permanent_upload/data/supported_file_types.csv

```diff
@@ -0,0 +1,45 @@
+wav,audio,ok,"wav,mp3"
```

Comment:
Please add a header row to this spreadsheet.

That'd also allow referencing the columns by name rather than index, by using [DictReader](https://docs.python.org/3/library/csv.html#csv.DictReader).